contracts/poap/src/state.rs

```diff
@@ -0,0 +1,123 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::{Addr, Timestamp};
+use cw_storage_plus::{Item, Map};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct Config {
+    pub admin: Addr,
```

Comment:
I think we can use [`cw_controllers::Admin`](https://github.com/CosmWasm/cw-plus/blob/main/packages/controllers/src/admin.rs#L29) to handle the admin state.
What do you think?

Reply:
That struct stores the admin as `Option<Addr>`, in our case the admin address is always defined and using that will just add boilerplate code to unwrap the address.